build.sh

```diff
@@ -121,6 +124,15 @@ if [ "${B_CORE}" = "y" ]; then
 	"./phoenix-rtos-build/build-core-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.sh"
 fi
 
+#
+# Install test in the phoenix-rtos filesystem
+#
+
+if [ "${B_TEST}" = "y" ] && [ "${TARGET_FAMILY}" = "ia32" ]; then
```

Comment:
target-specific logic should be put either in `_target/build.project.*` or in `build-core-*.sh` if it's a part of core

I think:
 - `all` should not make `test` target (`all` should build production-ready image, it will still compile the tests, so developer can add the individual tests to the production image if needed
 - `install` for tests should be done on all targets, we can probably add `TEST_MAKECMDGOALS` similar to `KERNEL_MAKECMDGOALS` to make it easy

Reply:
I moved a part of code into `phoenix-rtos-project`. I removed `test` from `all` target.